gzip_ecap_extended_compressible_types_v1_2.patch

```diff
@@ -0,0 +1,48 @@
+--- src/adapter_gzip.cc	Sun Jun 12 02:10:41 2016
++++ src/adapter_gzip.cc	Sun Jun 12 02:14:33 2016
+@@ -367,7 +367,8 @@
+ 
+ 	/**
+ 	 * Checks the Content-Type response header.
+-	 * At this time, only responses with "text/html" content-type are allowed to be compressed.
++	 * Added more real text types
++	 * Modified by Y.Voinov (c) 2016
+ 	 */
```

Comment:
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA256
 
Ok.

It is ok now?


12.06.2016 18:16, Constantin Rack пишет:
>
> In gzip_ecap_extended_compressible_types_v1_2.patch
<https://github.com/c-rack/squid-ecap-gzip/pull/17#discussion_r66720119>:
>
> > @@ -0,0 +1,48 @@
> > +--- src/adapter_gzip.cc    Sun Jun 12 02:10:41 2016
> > ++++ src/adapter_gzip.cc    Sun Jun 12 02:14:33 2016
> > +@@ -367,7 +367,8 @@
> > +
> > +     /**
> > +      * Checks the Content-Type response header.
> > +-     * At this time, only responses with "text/html" content-type
are allowed to be compressed.
> > ++     * Added more real text types
> > ++     * Modified by Y.Voinov (c) 2016
> > +      */
>
> You need to remove the lines, |commit --amend| and force push to your
|ptach-1| branch.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
<https://github.com/c-rack/squid-ecap-gzip/pull/17/files/50b61aad4922347d1e5509be78f044ea8fa1df55#r66720119>,
or mute the thread
<https://github.com/notifications/unsubscribe/AOUiqObYz9qUJO3OhmgjrkWtCVgGaCQaks5qK_iHgaJpZM4IzpAA>.
>

-----BEGIN PGP SIGNATURE-----
Version: GnuPG v2
 
iQEcBAEBCAAGBQJXXVPDAAoJENNXIZxhPexG5YUH+wRQ2BmKRnC02WtIPAhDjghi
Jvx6/MYshrkipNUHUWl2F/sCF16NbysZrj9P4byWaQKmL3GzMjpu5QKErZW6UYf0
uRZw88UyzNv5Q/se1DjNc1SKmUZ27U/yb1hzcrqBrdw2DnqRyCJ3xVPq7SebAH0g
7/LCS2i9tS9COG8LGZT8lJyLIU7RUe4kCT4VJWcXJim4/SBYkKPbKsVBJRPbJTgM
IZUGXK80NSb21ha/Hc8htIqQlpTOlHQcUgP04t4rVtexJUsZM/wLx/TRM4Fhoj7X
rKdm8o4U7ybs0kijbaDVi9WTvh9AR/NnsNFGjLXWMyWFDGhFjnW6n5zcOopWXbg=
=8vHP
-----END PGP SIGNATURE-----